examples/signature/App.js

```diff
@@ -1,72 +1,37 @@
-import Expo from 'expo';
 import * as ExpoPixi from 'expo-pixi';
-import React, { Component } from 'react';
-import { Image, Button, Platform, AppState, StyleSheet, Text, View } from 'react-native';
+import React from 'react';
+import { Button, StyleSheet, View } from 'react-native';
 
-const isAndroid = Platform.OS === 'android';
-function uuidv4() {
-  //https://stackoverflow.com/a/2117523/4047926
-  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
-    var r = (Math.random() * 16) | 0,
-      v = c == 'x' ? r : (r & 0x3) | 0x8;
-    return v.toString(16);
-  });
-}
-
-export default class App extends Component {
-  state = {
-    image: null,
-    strokeColor: 0,
-    appState: AppState.currentState,
-  };
-
-  handleAppStateChangeAsync = nextAppState => {
-    if (this.state.appState.match(/inactive|background/) && nextAppState === 'active') {
-      if (isAndroid && this.sketch) {
-        this.setState({ appState: nextAppState, id: uuidv4(), lines: this.sketch.lines });
-        return;
-      }
-    }
-    this.setState({ appState: nextAppState });
-  };
-
-  componentDidMount() {
```

Comment:
I believe removal of this will break switching apps in Android - the drawing will be cleared upon return to the app.